python/rikai/pytorch/models/fasterrcnn_resnet50_fpn.py

```diff
@@ -12,12 +12,28 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
+import torchvision
+
 from .torchvision import ObjectDetectionModelType
+from rikai.mixin import Pretrained
+from rikai.spark.sql.model import ModelSpec
+from rikai.spark.sql.codegen.dummy import DummyModelSpec
 
 __all__ = ["MODEL_TYPE"]
 
 
-class FasterRCNNModelType(ObjectDetectionModelType):
+class FasterRCNNModelType(ObjectDetectionModelType, Pretrained):
+    def load_pretrained_model(self):
+        return torchvision.models.detection.fasterrcnn_resnet50_fpn()
+
+    def load_model(self, spec: ModelSpec, **kwargs):
+        if isinstance(spec, DummyModelSpec):
```

Comment:
Can we do this dispatch in Rikai's model loader code instead of user-facing ModelType?